middlewares/isValidIReview.js

```diff
@@ -0,0 +1,10 @@
+import { isValidObjectId } from 'mongoose';
+import { HttpError } from '../helpers/HttpError.js';
+
+export const isValidReviewId = (req, res, next) => {
+  const { reviewId } = req.params;
```

Comment:
not the most flexible solution, to validate `dishId` you will need another middleware)
you can do something like this:

```
export const isValidId = (idParamName) => (req, res, next) => {
  const id = req.params[idParamName];
  ...
}
```

and pass it to route like this:
`app.post('/somePath', isValidId('reviewId'), (req, res) => {})`